### ISSUE TITLE
Update the title

### ISSUE THREAD

**Issue by bonsall** [snapappointments/bootstrap-select#1416]
It looks like i can't update the title without destroying the selectpicker and changing the html, like this.

```
$('.selectpicker').selectpicker('destroy');
$('select.selectpicker').html(someOptions);
$('.selectpicker').selectpicker({title = 'New Title'});
```

Did I miss something along the way? Can this be added as a new feature?

**Comment by caseyjhol** [comment]
This change will be a part of the v1.14.0 release. This issue will be updated after it's made available.

**Comment by caseyjhol** [comment]
Released in [v1.14.0-beta](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.14.0-beta)!
